### PR TITLE
Keep other 3th party repos enabled as much as possible

### PR DESCRIPTION
**Pull request by markVnl**
- Omit running signal-event software-repos-save explicitly

Hi Stephane, do not know if this is released or  simply is being forgotten...

**NOTE** : review this (for all other nethserverians i did rebuild and test, 
I think you know this better than me, so keep the review to you :)

### CODE REVIEW

File: createlinks [hunk 1]
```diff
@@ -5,12 +5,12 @@ use esmith::Build::CreateLinks  qw(:all);
 my $event = "nethserver-urbackup-update";
 event_actions($event,
              'initialize-default-databases' => '00',
```
Comment (stephdl) [inline createlinks]: please remove the comma at the end

File: createlinks [hunk 2]
```diff
@@ -5,12 +5,12 @@ use esmith::Build::CreateLinks  qw(:all);
...
-             'urbackup_enablerepo' => 10);
 
 foreach my $event (qw ( nethserver-urbackup-update nethserver-urbackup-save ))
     {
     templates2events("/etc/httpd/conf.d/default-virtualhost.inc",  $event);
     templates2events("/etc/httpd/conf.d/urbackup.conf",  $event);
+    templates2events("/etc/nethserver/eorepo.conf", $event);
```
Comment (stephdl) [inline createlinks]: `/etc/nethserver/eorepo.conf` can we only set this template to nethserver-urbackup-update (only for update and first install)

**Review by stephdl — CHANGES_REQUESTED** [review]

File: createlinks (hunk 2 above)
Comment (markVnl) [reply createlinks]: Yes do you prefer to split op the foreach ? Like:

```
my $event = "nethserver-urbackup-update";
event_actions($event,
     'initialize-default-databases' => '00')

templates2events("/etc/httpd/conf.d/default-virtualhost.inc",  $event);
templates2events("/etc/httpd/conf.d/urbackup.conf",  $event);
templates2events("/etc/nethserver/eorepo.conf", $event);
event_services($event, 'httpd' => 'reload');
event_services($event, 'urbackup-server' => 'restart');

my $event = "nethserver-urbackup-save";
event_actions($event,
    'firewall-adjust' => '10');

templates2events("/etc/httpd/conf.d/default-virtualhost.inc",  $event);
templates2events("/etc/httpd/conf.d/urbackup.conf",  $event);
event_services($event, 'httpd' => 'reload');
event_services($event, 'urbackup-server' => 'restart');
```

Comment (markVnl) [reply createlinks]: Or keep as is, like i did now?

Comment (stephdl) [reply createlinks]: good

**Review by stephdl — APPROVED** [review]

**Comment by stephdl** [comment]
released @markVnl

**Pull request merged**